include/swift/Basic/SourceManager.h

```diff
@@ -17,13 +17,16 @@
 #include "llvm/ADT/Optional.h"
 #include "llvm/ADT/StringMap.h"
 #include "llvm/Support/SourceMgr.h"
+#include "clang/Basic/FileManager.h"
+#include "swift/Basic/FileSystem.h"
```

Comment:
(here and elsewhere) The order here should go "swift", "clang", "llvm".

(Also, hopefully some day we can sink the Clang FileManager down to LLVM.)